adapters/syslog/syslog.go

```diff
@@ -23,9 +24,20 @@ const defaultRetryCount = 10
 var (
 	hostname         string
 	retryCount       uint
+	tcpFraming       TCPFraming
 	econnResetErrStr string
 )
 
+// TCPFraming represents the type of framing to use for syslog messages
+type TCPFraming string
+
+const (
+	// TraditionalTCPFraming is the traditional LF framing of syslog messages on the wire
+	TraditionalTCPFraming TCPFraming = "traditional"
+	// OctetCountedTCPFraming prepends the size of each message before the message. https://tools.ietf.org/html/rfc6587#section-3.4.1
+	OctetCountedTCPFraming = "octet-counted"
```

Comment:
Don't you miss TCPFraming type here?
`OctetCountedTCPFraming TCPFraming = "octet-counted"`

Reply:
Doh, yes I do; thanks for catching that!